pkg/kubelet/dockershim/network/cni/cni.go

```diff
@@ -68,6 +69,22 @@ type cniPortMapping struct {
 	HostIP        string `json:"hostIP"`
 }
 
+// cniBandwidthEntry maps to the standard CNI bandwidth Capability
+// see: https://github.com/containernetworking/cni/blob/master/CONVENTIONS.md and
+// https://github.com/containernetworking/plugins/blob/master/plugins/meta/bandwidth/README.md
+type cniBandwidthEntry struct {
+	// IngressRate is the bandwidth rate in bps for traffic through container. 0 for no limit. If ingressRate is set, ingressBurst must also be set
```

Comment:
`in bps` is ambiguous.  Please spell out "bits" or "bytes"

Reply:
bits per second. Done.